src/components/card/card.scss

```diff
@@ -0,0 +1,196 @@
+
+@import "variables";
+@import "shadows";
+@import "default-theme";   // TODO: Remove this
+
+$md-card-default-padding: rem(2.400) !default;
+$md-card-mobile-padding: rem(2.400) rem(1.600) !default;
+$md-card-border-radius: 2px !default;
+$md-card-header-size: rem(4.000) !default;
+
+md-card {
+  display: block;
+  position:relative;
+  padding: $md-card-default-padding;
+  border-radius: $md-card-border-radius;
+  box-shadow: $md-shadow-bottom-z-1;
+  font-family: $font-family;
+  background: md-color($md-background, 50);       // temp until 0 works
+}
+
+%md_card_section_base {
+  display: block;
+  margin-bottom: rem(1.600);  
+}
+
+md-card-title {
+  @extend %md_card_section_base;
+  font-size: rem(2.400);
+  font-weight: 400;
+}
+
+md-card-subtitle {
+  @extend %md_card_section_base;
+  font-size: $md-body-font-size-base;
+  color: md-color($md-foreground, "secondary-text");
+}
+
+md-card-content {
+  @extend %md_card_section_base;
+  font-size: $md-body-font-size-base;
+}
+
+md-card-actions {
+  @extend %md_card_section_base;
+  margin-left: rem(-1.600);
+  margin-right: rem(-1.600);
+  padding: rem(0.800) 0;
+}
+
+img[md-card-image] {
```

Comment:
These can just be `[md-card-image]` without being coupled to the `img` element. This didn't occur to me yesterday, but it would be totally valid for someone to use a different element type here, such as [`<picture>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture).